src/canonicalizeActionName.ts

```diff
@@ -0,0 +1,32 @@
+import { ActionType } from "./typings/Types";
+
+const actionAliasToCanonicalName: Record<string, ActionType> = {
```

Comment:
@AndreasArvidsson for now I'm just mapping from new to old.  Should probably switch over the action names here eventually, but not sure it's worth the effort for this initial PR.  Fwiw, the new action names are not actually tested when using this approach